app/services/get-export-csv.js

```diff
@@ -40,6 +40,9 @@ const WORKLOAD_PERCENTAGE_BREAKDOWN_EXPORT_FIELDS = ['regionName', 'lduName', 't
 const SUSPENDED_LIFERS_EXPORT_FIELD_NAMES = ['Region Name', 'Probation Delivery Unit', 'Team Name', 'Tier Code', 'Row Type', 'CRN', 'Case Type', 'Offender Manager Name', 'Grade Code', 'In Custody?', 'Register Level', 'Register Category', 'Register Category Description', 'Registration Date']
 const SUSPENDED_LIFERS_EXPORT_FIELDS = ['regionName', 'lduName', 'teamName', 'tierCode', 'rowType', 'caseReferenceNo', 'caseType', 'offenderManagerName', 'gradeCode', 'inCustody', 'registerLevel', 'registerCategory', 'registerCategoryDescription', 'registrationDate']
 
+const T2A_EXPORT_FIELD_NAMES = ['Region Name', 'Region Id', ' lduName', 'lduId', 'TeamN ame', 'Team Id', 'CRN', 'Workload Owner Id', 'Om Name', 'Om Code', 'Event Number', 'Allocation Date', 'NSI Outcome Code', 'NSI Outcome Descript']
```

Comment:
Could the following changes be made please:

- Remove the ID fields
- Change lduName to Probation Delivery Unit
- Change TeamN ame to Team Name
- Change Om Name to Offender Manager Name
- Change Om Code to Grade Code
- Change NSI Outcome Descript to NSI Outcome Description

Thanks